src/containers/search/style.scss

```diff
@@ -83,6 +83,14 @@ td > .detailContainer {
 .agrDownloadBtn {
 }
 
+.relatedDataLinks {
+  margin-left: 1em;
+}
+
+.relatedDataLinks ul {
+  list-style-type: none;
+}
+
```

Comment:
SASS THIS!!
```
.relatedDataLinks {
  margin-left: 1em;

  ul {
    list-style-type: none;
  }
}
```
Also, isn't `.list-unstyled` already handling the `list-style-type` for you?